From 1de59ade083fc405e603b8c2f0c4e8e0ff7c65d6 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solusos.com>
Date: Sun, 8 Sep 2013 10:12:15 +0100
Subject: [PATCH] Change the default admin group to sudo

SolusOS doesn't use the wheel group, members of the sudo group are considered
to be system administrators and as such are granted all privileges associated
with this role. This patch changes the default unix group of administrators
to sudo.
---
 src/polkitbackend/50-default.rules |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/polkitbackend/50-default.rules b/src/polkitbackend/50-default.rules
index f427ae1..1f038fc 100644
--- a/src/polkitbackend/50-default.rules
+++ b/src/polkitbackend/50-default.rules
@@ -8,5 +8,5 @@
 // about configuring polkit.
 
 polkit.addAdminRule(function(action, subject) {
-    return ["unix-group:wheel"];
+    return ["unix-group:sudo"];
 });
-- 
1.7.10.4

